### PR TITLE
EOS-14685: CSM shows raid_data| fault message

### PR DESCRIPTION
**Pull request by shindeam**
## Problem Statement
<pre>
  <code>
    Story Ref (if any):
   beta2: renovo: "LDR R1" RC10: CSM shows "raid_data | fault" message.
  </code>
</pre>
## Problem Description
<pre>
  <code>
    MDRAID fault alerts are seen without inducing faults.
  </code>
</pre>
## Solution
<pre>
  <code>
    Fix is present  in GA & main branches, Added the patch and created PR against the Beta branch
  </code>
</pre>
## Sanity testing on RPM done
<pre>
  <code>
    - [ ] Yes
    - [ ] No
  </code>
</pre>
## Unit/Manual Testing Description
<pre>
  <code>
    Please describe the tests that you ran to verify your changes.
  </code>
</pre>

### CODE REVIEW

File: jenkins/build.sh [hunk 1]
```diff
@@ -131,11 +129,8 @@ mkdir -p ${RPM_BUILD_PATH}/SOURCES
 TOPDIR=$(realpath $RPM_BUILD_PATH)
 echo $TOPDIR
 
-tar -czvf ${RPM_BUILD_PATH}/SOURCES/systemd-python36-${VERSION}.tgz -C ${BASE_DIR}/.. sspl/systemd-python36
-rpmbuild --define "version $VERSION" --define "git_rev $GIT_VER" --define "_topdir $TOPDIR" -bb $BASE_DIR/systemd-python36/systemd-python36.spec
-
 echo 'Installing systemd-python36*...'
-yum install -y ${RPM_BUILD_PATH}/RPMS/x86_64/systemd-python36-*
+yum install -y systemd-python36-*
```
Comment (cdeshmukh) [inline jenkins/build.sh:133]: some unrelated changes i see in this PR

File: low-level/sensors/impl/generic/raid.py [hunk 2]
```diff
@@ -390,8 +394,14 @@ def _process_missing_md_devices(self, md_device_list, drive_dict):
         for line in raid_conf_data:
             try:
                 raid_conf_field = line.split(" ")
-                if "md" in raid_conf_field[1]:
-                    conf_device_list.append(raid_conf_field[1])
+                if "#" not in raid_conf_field[0] and "ARRAY" in raid_conf_field[0] and \
```
Comment (cdeshmukh) [inline low-level/sensors/impl/generic/raid.py:397]: can you elaborate how this change is fixing the issue ?

**Comment by indrajitzagade** [comment]
> 
> 
> This issue/pull request has been marked as `needs attention` as it has been left pending without new activity for 4 days. Tagging @indrajitzagade @thavanathan for appropriate assignment. Sorry for the delay & Thank you for contributing to CORTX. We will get back to you as soon as possible.

@shindeam Please work on review comments and get this to a closure.
